Directory.Build.targets

```diff
@@ -1,8 +1,11 @@
 <Project>
   <PropertyGroup>
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.2' ">$(MicrosoftNETCoreApp22PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
revert the changes in this file.

Reply:
Actually if I read the the comment [here](https://github.com/aspnet/AuthSamples/pull/49/files#diff-c06bb39fa761f879be361a13a9c1fd92R7) right we don't need anything in this PR. Closing.